wordle-web/pages/game.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <v-card
+    class="mx-auto my-8 w-75 pa-5 rounded"
+    color="secondary text-primary"
+    elevation="4"
+  >
+    <v-card-item class="text-center">
+      <v-card-title class="font-weight-bold"
+        >Pink Wordle <v-icon icon="mdi-heart"></v-icon
+      ></v-card-title>
+      <v-card-subtitle>
+        This is our super basic Pink Wordle game! Good luck!
+      </v-card-subtitle>
+      <v-card-text class="font-italic">
+        "Pink Wordle is a delightful twist on the classic word puzzle game,
+        Wordle. In this charming variant, the game's interface is adorned with
+        various shades of pink, offering a refreshing and visually pleasing
+        experience to the players. The objective remains the same - guess the
+        hidden word within six attempts.Pink Wordle combines the mental
+        challenge of the original game with a unique aesthetic, making it a
+        must-try for puzzle enthusiasts and pink lovers alike."
+      </v-card-text>
+    </v-card-item>
+    <v-spacer />
+
+    <v-card-item
+      class="justify-center"
+      v-for="(guess, i) of game.guesses"
+      :key="i"
+    >
+      <v-chip
+        v-for="(letter, j) of guess.letters"
+        :key="j"
+        :color="displayLetterState(letter)"
+        class="ma-1 pa-2 text-h5"
+        >{{ letter.char }}
+      </v-chip>
+    </v-card-item>
+
+    <v-card-text class="text-center font-weight-bold text-h6 my-2">
+      {{ displayGameState() }}
+    </v-card-text>
+    <v-text-field
+      v-model="myGuess"
+      label="Enter your guess"
+      outlined
+      clearable
+      :disabled="game.state === GameState.Playing ? false : true"
```

Comment:
I think this could be simplified to "game.state !== GameState.Playing"

Reply:
Good point, I will fix this and go through the code since I'm pretty sure I used this same logic in another place